### PR TITLE
Change color for submit form

### PR DESCRIPTION
**Pull request by David-Verde**
create the contact form
collaboration os my partner to fill out a form.

### CODE REVIEW

File: work.css [hunk 1]
```diff
@@ -7,7 +7,7 @@
   margin: 0;
   padding: 0;
   box-sizing: border-box;
-  font-family: 'Poppins' sans-serif;
+  font-family: 'Poppins', sans-serif;
 }
```
Comment (Mre55) [inline work.css]: Here in your form let's add `maxlength` attribute to limit the name and input message fields as in the project [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md#project-requirements). Also kindly, add `required` attribute for all inputs to activate the validation on submit.


image

File: work.css [hunk 2]
```diff
@@ -7,7 +7,7 @@
...
 
 img {
```
Comment (Mre55) [inline work.css:6]: Also, please as mentioned in the project's requirement stick to the Figma as much as possible. Let's add a background image from the .css file for the contact form section (not `img` tag from .html file). Also, add the curve at the left top corner for the contact section. And change the name of the button `Submit` to `Get In Touch`.

| Your Design | Figma Design |
|-------------|---------------|
| image | image |

**Review by Mre55 — CHANGES_REQUESTED** [review]
**Hi @Unyielding1,**

You have done a great job so far! 💪 👏 
You still need to work on a few issues before moving on to the next project, but you're almost there!

## To highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- Linters are working properly ✅ 

## Required Changes ♻️🛠️ 
_Check the comments under the review._

- [ ] You have separate `.html` file for `headline` section and `work and contact form` section. 
image 

Please move all the content which is inside the `work.html` file to the `index.html` file after the end of `<section id="background">`.

- [ ]  Kindly use this [README.md file template](https://github.com/microverseinc/readme-template/blob/master/README.md) by copying everything from that and editing the content for your particular project with your credentials.





## Optional suggestions 🟡 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -37,46 +38,265 @@
...
+    </div>
+  </article>
+  <article class="contact">
+    <div class="grid-container">
+      <div class="grid-item grid-item-1 flex-container contact_flex">
+        <h2 class="contact_header flex-item-1">Contact Me</h2>
+        <p class="contact_paragraph flex-item-2">
+          If you have an application you are interested in developing, a
+          feature that you need built or a project that needs coding. I’d love
+          to help with it
```
Comment (codepantha) [inline index.html:287]: - [ ] As the previous reviewer suggested, please stick to the Figma design as much as possible. Let's add a background image from the .css file for this section
### Your Design
image

### Figma Design
image

**Review by codepantha — CHANGES_REQUESTED** [review]
Hi @Unyielding1 👋, 

## STATUS: REQUESTED CHANGES ♻️

Great work on making the changes requested by the previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Required Changes ♻️
_Check the comments under the review._

- [ ] As requested by the previous reviewer, kindly adapt this [README.md file template](https://github.com/microverseinc/readme-template/blob/master/README.md) to create a professional README for this project.


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by David-Verde** [comment]
Yeah, you're right I forget to fix the Readme 👍  
image
mmm please let me know if you still can´t visualize the CSS background it is weird last time y did the pull request was visible and this time on my PC too
image

**Review by CollinsTatang — APPROVED** [review]
# Approved 👏  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**